src/main/java/knightminer/inspirations/library/InspirationsRegistry.java

```diff
@@ -41,15 +44,30 @@
 public class InspirationsRegistry {
 	public static final Logger log = Util.getLogger("api");
 
+	// Blocks with this tag act as fire for the cauldron.
+	public static final Tag<Block> TAG_CAULDRON_FIRE = new BlockTags.Wrapper(new ResourceLocation(Inspirations.modID, "cauldron_fire"));
+	// Items with this tag are registered to have fluid tank functionality.
+	public static final Tag<Item> TAG_DISP_FLUID_TANKS = new ItemTags.Wrapper(new ResourceLocation(Inspirations.modID, "fluid_containers"));
```

Comment:
Not really sure about that and the rest of the dispenser functionality. Without registering something on every item or futzing with the map whenever the config changes, not sure if you can do anything and be compatible here...

Reply:
I want to avoid registering something on every item, but not sure what other choice I have. Might have to hold off on the whole dispenser automation for now